static/src/stylesheets/module/commercial/glabs/_hosted-header.scss

```diff
@@ -13,6 +13,26 @@
     }
 }
 
+.has-no-flex .hosted__header {
+    .hostedbadge {
+        position: absolute;
+        top: 0;
```

Comment:
Maybe it would be better to extract the common styles `position: absolute; top: 0;` to a mixin?